### PR TITLE
testsuite: test dumpdir owner and mode

### PR DESCRIPTION
**Pull request by jfilak**
Signed-off-by: Jakub Filak <jfilak@redhat.com>

### CODE REVIEW

File: tests/runtests/ccpp-plugin/runtest.sh [hunk 1]
```diff
@@ -45,8 +45,9 @@ rlJournalStart
         pushd $TmpDir
     rlPhaseEnd
 
-    rlPhaseStartTest "CCpp plugin works"
-        generate_crash
+    rlPhaseStartTest "CCpp plugin (testuser crash)"
+	useradd testuser
```
Comment (sorki) [inline tests/runtests/ccpp-plugin/runtest.sh]: whitespace errors ^^

File: tests/runtests/ccpp-plugin/runtest.sh [hunk 2]
```diff
@@ -66,6 +72,7 @@ rlJournalStart
 
     rlPhaseStartCleanup
         rlRun "abrt-cli rm $crash_PATH" 0 "Remove crash directory"
+	userdel -e testuser
```
Comment (sorki) [inline tests/runtests/ccpp-plugin/runtest.sh]: and here

**Comment by sorki** [comment]
Otherwise good.

**Comment by jfilak** [comment]
Thanks. I have pushed this version : https://github.com/abrt/abrt/commit/3f2f8a81716fa9467a1025936ad52507bc2d3343

**Pull request closed without merging**